src/cohortMembers/dto/cohortMember-update.dto.ts

```diff
@@ -60,7 +60,8 @@ export class CohortMembersUpdateDto {
     type: String,
     description: "The status change reason",
   })
-  @IsOptional()
+  @ValidateIf(o => o.memberStatus === MemberStatus.DROPOUT)
+  @IsString({ message: 'statusReason can not empty if member status is dropout' })
```

Comment:
@poojakarma change this to "Reason is mandatory as you are dropping out this member."

Reply:
column name is memberStatus or status?

Reply:
memberStatus